examples/2_function_calling_fewshot.py

```diff
@@ -2,8 +2,9 @@
 import os
 from typing import Annotated
 
-from kani import AIParam, ChatMessage, FunctionCall, Kani, ai_function, chat_in_terminal
+from kani import AIParam, ChatMessage, Kani, ai_function, chat_in_terminal
 from kani.engines.openai import OpenAIEngine
+from kani.models import ToolCall
```

Comment:
`ToolCall` is part of `kani.models` and `FunctionCall` is part of `kani`. Is there a good reason for this?

EDIT: Seems like later down `FunctionCall` is also imported from `.models` so maybe there's just an inconsistency...

Reply:
Oops, this is just a result of pycharm's autoimport not resolving to the one from the root